src/nvim/api/keysets.lua

```diff
@@ -52,6 +52,7 @@ return {
     "force";
     "keepscript";
     "nargs";
+    "preview";
```

Comment:
Let's name it `cmdpreview` here and everywhere. That allows unambiguous grepping, discussions, discoverability, etc..

Reply:
I think when we use `nvim_create_user_command`, we're already working in the context of commands, so the "cmd" in "cmdpreview" inside `nvim_create_user_command` feels redundant to me